experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -1634,6 +1645,7 @@ def _aten_arange(
   requires_grad=False,
   device=None,
   pin_memory=False,
+  **kwargs
 ):
```

Comment:
do we need add `**kwargs` here to accept `**kwargs` from `res2 = func(input2, *args2, **kwargs2)`?

I seems met one situation that: func in `jaten.py` file didn't received `kwargs2` from `res2 = func(input2, *args2, **kwargs2)`

Reply:
yeah this is not needed. removed.